source/Octopus.Cli/Commands/Package/PushMetadataCommand.cs

```diff
@@ -21,13 +22,14 @@ public PushMetadataCommand(IOctopusAsyncRepositoryFactory repositoryFactory, IOc
             options.Add("package-id=", "The ID of the package; e.g. MyCompany.MyApp", v => PackageId = v);
             options.Add("version=", "The version of the package; defaults to a timestamp-based version", v => Version = v);
             options.Add("metadata-file=", "Octopus Package metadata Json file.", file => MetadataFile = file);
-            options.Add("replace-existing", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed. You can pass this flag to overwrite the existing package.", replace => ReplaceExisting = true);
+            options.Add("overwrite-mode=", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed (FailIfExists). You can use the overwrite mode to OverwriteExisting or IgnoreIfExists.", mode => OverwriteMode = (OverwriteMode)Enum.Parse(typeof(OverwriteMode), mode));
```

Comment:
I'd strongly prefer the options to be case-insensitive, like most Octo commands and parameters:
```suggestion
            options.Add("overwrite-mode=", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed (FailIfExists). You can use the overwrite mode to OverwriteExisting or IgnoreIfExists.", mode => OverwriteMode = (OverwriteMode)Enum.Parse(typeof(OverwriteMode), mode, true));
```